README.md

```diff
@@ -57,6 +57,10 @@ to learn about other command-line flags usable to configure the packaged app.
 To have high-resolution icons used by default for an app/domain, please
 contribute to the [icon repository](https://github.com/nativefier/nativefier-icons)!
 
+### Build Commands Library
+
+For a list of build commands contributed by the nativefier community take a look at the [CATALOG.md file](CATALOG.md)
```

Comment:
This is pretty cool! I didn't realize that you could direct link to other .md files like this. We should probably fix all our other links to be like this. 😆

Reply:
to be honest I didn't either. I just saw that it was done in the readme file...